serveradmin/serverdb/migrations/0012_migrate_change_tables.py

```diff
@@ -0,0 +1,139 @@
+# Generated by Django 3.2.16 on 2022-10-24 13:39
+
+import json
+
+from django.db import migrations, transaction
+from django.utils.timezone import now
+
+
+BATCH_SIZE = 2500
+
+
+def migrate_change_add(apps, schema_editor):
+    Change = apps.get_model('serverdb', 'Change')
+    ChangeAdd = apps.get_model('serverdb', 'ChangeAdd')
+
+    print('')  # Print on next line of migration output
+    print('\t- Starting migrating ChangeAdd objects.')
+
+    total = ChangeAdd.objects.count()
+    migrated = 0
+    while ChangeAdd.objects.count() > 0:
+        start_time = now()
+
+        with transaction.atomic():
+            changes = ChangeAdd.objects.all()[:BATCH_SIZE]
```

Comment:
Obsolete because of changing to direct SQL execution https://github.com/innogames/serveradmin/pull/294/commits/371a8c597cb85b5599472638c248d373d24d5b3f